### PR TITLE
Add translations for security questions

### PR DESCRIPTION
**Pull request by pcraig3**
Added the french translations for the following URLs:

- `/login/securityQuestions`
- `/login/securityQuestions2`
- `/login/questions/child`
- `/login/questions/addresses`
- `/login/questions/dateOfResidence`
- `/login/questions/prison`
- `/login/questions/bankruptcy`
- `/offramp/securityQuestion`

Still have a few edge cases, but this is good for now.

Need to have @EliseKa look it over.

### CODE REVIEW

**Comment by pcraig3** [comment]
Hi Élise, 

There are some translations missing:

- Please enter your child’s last name 
  - (on https://cra-claim-tax-benefits-pr-213.herokuapp.com/login/questions/child?lang=fr, if you just click "Continuer" without putting data in)
- Please select which date you are providing 
  - (on https://cra-claim-tax-benefits-pr-213.herokuapp.com/login/questions/prison, if you just click "Continue" without putting data in)
- Please enter your trustee's last name
  - (on https://cra-claim-tax-benefits-pr-213.herokuapp.com/login/questions/bankruptcy, if you just click "Continue" without putting data in)
- This page is currently under construction
- We are currently in the process of building this page. You can still proceed through your session by clicking Continue
  - (on https://cra-claim-tax-benefits-pr-213.herokuapp.com/login/questions/temp)

And we also have this page, but we're not planning to test it, so we don't have to do it right now.
https://cra-claim-tax-benefits-pr-213.herokuapp.com/login/questions/trillium

File: locales/fr.json [hunk 1]
```diff
@@ -314,30 +313,29 @@
...
-  "Details of your contribution to a Tax Free Savings Account (TFSA)": "Details of your contribution to a Tax Free Savings Account (TFSA)",
-  "Details of your Canada Child Benefit (CCB)": "Details of your Canada Child Benefit (CCB)",
-  "Date and amount of your most recent payment to CRA": "Date and amount of your most recent payment to CRA",
-  "I can’t answer these questions either": "I can’t answer these questions either",
-  "Answer a question from the list below": "Answer a question from the list below",
+  "Enter the last two mailing addresses before your current address": "Vos deux anciennes adresses postales les plus récentes",
+  "You must have used both addresses to file taxes. Choose a different question if you haven’t filed taxes from at least two other addresses.": "Vous devez avoir utilisé ces deux adresses postales pour produire une déclaration de revenus. Choisissez une autre question si vous n’avez pas produit de déclaration de revenus à ces deux adresses.",
+  "Mailing address 1": "Adresse postale 1",
+  "Street address": "Addresse de la rue",
```
Comment (EliseKa) [inline locales/fr.json]: This should be "Adresse municipale" in French.

File: locales/fr.json [hunk 2]
```diff
@@ -348,14 +346,16 @@
   "Using CRA software called": "En utilisant le logiciel de l'ARC,",
   "NETFILE": "IMPÔTNET",
   "For more information, contact CRA by:": "Pour de plus amples renseignements, communiquez avec l’ARC par :",
-  "Phone: 1-800-959-8281": "Phone: 1-800-959-8281",
-  "TTY: 1-800-665-0354": "Téléphone : 1-800-665-0354",
+  "Phone: 1-800-959-8281": "Téléphone : 1-800-959-8281",
```
Comment (EliseKa) [inline locales/fr.json]: You might want to double-check, but from what I saw on the website (https://www.canada.ca/fr/agence-revenu/organisation/coordonnees/numeros-telephone.html), the phone is different for French: 1-800-959-7383

File: locales/fr.json (hunk 1 above)
Comment (pcraig3) [reply locales/fr.json]: Good catch 👍

**Pull request merged**